### PR TITLE
DDF-3707 npm upgrades Express, Request, Phantomjs-prebuilt

### PR DESCRIPTION
**Pull request by shaundmorris**
#### What does this PR do?
Upgrade Express from 4.14.0 to 4.16.3 
Upgrade Request from 2.79.0 to 2.85.0
Upgrade Phantomjs-prebuilt from 2.1.14 to 2.1.16
#### Who is reviewing it? 
@djblue @andrewkfiedler 
(please choose AT LEAST two reviewers that need to approve the PR before it can get merged)
#### Select relevant component teams: 
@codice/ui
#### Choose 2 committers to review/merge the PR. 
(please choose ONLY two committers from below, delete the rest)
@clockard 
@bdeining
#### How should this be tested? (List steps with links to updated documentation)
Manual test of UIs upgraded
#### Any background context you want to provide?
NSP check will pass after upgrades are merged
#### What are the relevant tickets?
[DDF-3707](https://codice.atlassian.net/browse/DDF-3707)
#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

---
This change is [Reviewable](https://reviewable.io/reviews/codice/ddf/3099)

### CODE REVIEW

File: ui/packages/ace/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ace",
-  "version": "1.0.0",
+  "version": "1.3.3",
```
Comment (shaundmorris) [inline ui/packages/ace/package.json]: Whoops, didn't mean to update this version, not sure when we do though

**Comment by shaundmorris** [comment]
Manually tested the modules that had updated packages

**Comment by shaundmorris** [comment]
build now

**Comment by shaundmorris** [comment]
build now

File: ui/packages/ace/.nsprc [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+  "exceptions": [
+    "https://nodesecurity.io/advisories/566"
```
Comment (pklinef) [inline ui/packages/ace/.nsprc]: ❓ Can we leave a comment to why we added exceptions?

Comment (shaundmorris) [reply ui/packages/ace/.nsprc]: @pklinef  Good question. I asked the same thing. I'm not aware of way to add comments in JSON particularly for this format. I've only seen hacky ways of making them a field/value. But I would like to add it if we are aware of a way to do it. I wonder if we can track of it elsewhere if not directly in this file.

Comment (shaundmorris) [reply ui/packages/ace/.nsprc]: @pklinef I tried it out again, but remember why now we couldn't. When I add a comment it is like the file fails to get parsed and the exceptions are no longer read. Looks like comments worked at one point and doesn't anymore: https://github.com/nodesecurity/nsp/issues/207#issuecomment-378594564

**Comment by shaundmorris** [comment]
build was unrelated to install profiles

build now

**Pull request merged**